content/en/continuous_integration/setup_tests/javascript.md

```diff
@@ -123,6 +125,43 @@ Run your tests as you normally do, specifying the environment where test are bei
 DD_ENV=ci npm test
 {{< /code-block >}}
 
+{{% /tab %}}
+
+{{% tab "Cypress" %}}
+
+1. Set [`pluginsFile`][1] to `"dd-trace/cypress/plugin"`, for example through [`cypress.json`][2]:
+{{< code-block lang="json" filename="cypress.json" >}}
+{
+  "pluginsFile": "dd-trace/cypress/plugin"
+}
+{{< /code-block >}}
+
+If you've already defined a `pluginsFile`, you can still initialise the instrumentation with:
```

Comment:
Not sure what Data Dog uses but `initialise` is the British spelling, the American spelling would be `initialize`

Reply:
yeah good catch. I always wonder myself 😆 . They seem to use the American spelling 👍 . Thanks!